### DISCUSSION TITLE
How do I change the base colors of NEXTUI?

### DISCUSSION THREAD

**Discussion by Nicolas11Garcia** [heroui-inc/heroui#2176] [Q&A]
Does anyone know how I can change the color of these colors? What I want is that when entering color='danger' another color is displayed and not the default pink

![Captura de pantalla 2023-12-27 225749](https://github.com/nextui-org/nextui/assets/74791232/73ebae16-3274-4ee1-a614-5d191ddc2816)
![Captura de pantalla 2023-12-27 225814](https://github.com/nextui-org/nextui/assets/74791232/31f33c76-7a66-4391-9bfd-0a0bad9382fd)

**Comment by soxamz** [comment]
```jsx
//tailwind.config.js
 module.exports = {
  theme: {
    extend: {
      colors: {
        'danger': '#hex_color_code',
      }
    }
  },
  variants: {},
  plugins: [],
} 
```
Refer the link provided in order to know more : [nextui.org/docs/customization/colors](https://nextui.org/docs/customization/colors)
Hope it helps!

**Reply by Nicolas11Garcia** [reply]
Doesnt work:(

**Reply by choonho** [reply]
not in theme, add in plugins

```
  plugins: [
    nextui({
      themes: {
        light: {
          colors: {
            primary: '#2E1065',
          },
        }
      }
  })],
}
```
